huaweicloud/services/acceptance/er/resource_huaweicloud_er_instance_test.go

```diff
@@ -91,13 +95,34 @@ func TestAccInstance_basic(t *testing.T) {
 	})
 }
 
-func testInstance_basic(name string, bgpAsNum int) string {
+func testInstance_basic_step1(name string, bgpAsNum int) string {
 	return fmt.Sprintf(`
 resource "huaweicloud_er_instance" "test" {
   availability_zones = ["%[1]s"]
 
   name = "%[2]s"
   asn  = %[3]d
+
+  tags = {
+    foo = "bar"
+    key = "value"
+  }
+}
+`, acceptance.HW_AVAILABILITY_ZONE, name, bgpAsNum)
```

Comment:
Maybe add a preCheck for `HW_AVAILABILITY_ZONE`

Reply:
done.